DiscordWikiBot/.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
On a related note, I am somewhat sceptical that adding Dependabot would be helpful (since any suggested updates would be mostly non-critical). While no changes in your pull request are required because I can always remove it later, a pull request should generally try to fit a specific purpose (specified by the PR title) so it is easier for maintainers to manage.